### DISCUSSION TITLE
Automatic search in the media folder for images

### DISCUSSION THREAD

**Discussion by BenderVano** [damionx7/Zettel-Notes-Documentation#128] [Feature Requests]
Obsidian adds an image in this form ![[IMG_20240706_142715.jpg]] ("short form"), but I put all media files in the "assets" folder and obsidian automatically finds them inside the folder. 
It is required that the application automatically finds images inside the media folder without specifying it in the tag. 
Now it only works if I manually specify a subfolder: ![[assets/IMG_20240706_142715.jpg]]

**Comment by dhakkann** [comment]
The app supports this feature for wikilinks for notes, but it doesn't work for images.
This is the major thing missing I am missing.
Would really love to have this.
